Formula/wine.rb

```diff
@@ -9,21 +9,16 @@ class Wine < Formula
   homepage "https://www.winehq.org/"
 
   stable do
-    url "https://dl.winehq.org/wine/source/3.0/wine-3.0.4.tar.xz"
-    mirror "https://downloads.sourceforge.net/project/wine/Source/wine-3.0.4.tar.xz"
-    sha256 "d45a88edbe7db363e297cacedc94e66df3464504a01c2eccf04f518066a6fb0c"
+    url "https://dl.winehq.org/wine/source/4.0/wine-4.0.tar.xz"
+    mirror "https://downloads.sourceforge.net/project/wine/Source/wine-4.0.tar.xz"
+    sha256 "6736cdee95b2b8bb021ec0c19497ed8cad5ae2c8bfdb7ab5dc687ff92a480d4d"
 
     # Patch to fix screen-flickering issues. Still relevant on 3.0.
     # https://bugs.winehq.org/show_bug.cgi?id=34166
     patch do
       url "https://raw.githubusercontent.com/Homebrew/formula-patches/74c2566/wine/2.14.patch"
       sha256 "6907471d18996ada60cc0cbc8462a1698e90720c0882846dfbfb163e5d3899b8"
     end
-
-    resource "mono" do
-      url "https://dl.winehq.org/wine/wine-mono/4.7.3/wine-mono-4.7.3.msi"
```

Comment:
According to https://source.winehq.org/git/wine.git/blob/refs/tags/wine-4.0:/dlls/appwiz.cpl/addons.c the correct Mono version for Wine 4.0 is 4.7.5. Please update the mono resource, and the URL at line 6 of the formula.

Reply:
https://github.com/Homebrew/homebrew-core/pull/36323/files#diff-129c0a4eb28c8c04cf7a1f694e1633e6R46 for the mono update